pallets/schemas/src/lib.rs

```diff
@@ -183,11 +182,5 @@ pub mod pallet {
 			}
 			None
 		}
-
-		pub fn calculate_schema_cost(schema: Vec<u8>) -> Weight {
-			let schema_len = schema.len() as u32;
-			let schema_weight = T::WeightInfo::register_schema(schema_len);
-			schema_weight
-		}
```

Comment:
Move it away from schema to tx pallet where we will have all custom configs for MRC related costs